### PR TITLE
189 output 64-bit netcdf files

### PR DESCRIPTION
**Pull request by scotthavens**
Added an option to output SMRF netcdf files as float (32-bit) or double (64-bit). The default is float (32-bit) and the option for double will be used in AWSM when comparing all the different methods.

### CODE REVIEW

File: smrf/tests/output/test_netcdf.py [hunk 1]
```diff
@@ -10,14 +11,11 @@
 
 class TestOutputNetCDF(SMRFTestCase):
 
-    @classmethod
-    def setUpClass(cls):
+    def setUp(self):
         super().setUpClass()
-        cls.smrf = SMRF(cls.config_file)
+        self.smrf = SMRF(self.config_file)
```
Comment (jomey) [inline smrf/tests/output/test_netcdf.py:16]: Why did you need to switch to an instance here?

Comment (scotthavens) [reply smrf/tests/output/test_netcdf.py:16]: It gets initialized with the 64-bit test first and doesn't get overwritten. So a new instance was needed to set all the config
